src/prog_models/prognostics_model.py

```diff
@@ -405,6 +405,51 @@ def next_state(self, x, u, dt) -> dict:
         dx = self.dx(x, u)
         return {key: x[key] + dx[key]*dt for key in dx.keys()}
 
+    def __next_state(self, x, u, dt) -> dict:
+        """
+        State transition equation: Calculate next state and add noise
+
+        Parameters
+        ----------
+        x : dict
+            state, with keys defined by model.states \n
+            e.g., x = {'abc': 332.1, 'def': 221.003} given states = ['abc', 'def']
+        u : dict
+            Inputs, with keys defined by model.inputs \n
+            e.g., u = {'i':3.2} given inputs = ['i']
+        dt : number
+            Timestep size in seconds (≥ 0) \n
+            e.g., dt = 0.1
+
+        Returns
+        -------
+        x : dict
+            Next state, with keys defined by model.states
+            e.g., x = {'abc': 332.1, 'def': 221.003} given states = ['abc', 'def']
+
+        Example
+        -------
+        | m = PrognosticsModel() # Replace with specific model being simulated
+        | u = {'u1': 3.2}
+        | z = {'z1': 2.2}
+        | x = m.initialize(u, z) # Initialize first state
+        | x = m.__next_state(x, u, 0.1) # Returns state, with noise, at 3.1 seconds given input u
+        
+        See Also
+        --------
+        next_state
+
+        Note
+        ----
+        A model should not overwrite '__next_state'
+        """
+        
+        # Calculate next state
+        self.next_state(x, u, dt)
```

Comment:
Here you never actually save the next state. 

It should be
```suggestion
        X = self.next_state(x, u, dt)
```